meta-resin-common/recipes-containers/docker/docker/docker.service

```diff
@@ -7,6 +7,7 @@ Requires=docker.socket var-lib-docker.mount
 [Service]
 Type=notify
 Restart=always
+ExecStartPre=/bin/sh -c "/usr/bin/test ! -s /etc/docker/key.json && /bin/rm /etc/docker/key.json"
```

Comment:
I think it would be clearer if it would be changed to something like 
`/usr/bin/test -s /etc/docker/key.json || /bin/rm /etc/docker/key.json`